cloud/deployment/workerqueue/workerqueue.go

```diff
@@ -0,0 +1,223 @@
+package workerqueue
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"sort"
+	"strconv"
+
+	"github.com/astronomer/astro-cli/astro-client"
+	"github.com/astronomer/astro-cli/cloud/deployment"
+	"github.com/astronomer/astro-cli/pkg/input"
+	"github.com/astronomer/astro-cli/pkg/printutil"
+)
+
+var (
+	ErrWorkerQueueDefaultOptions = errors.New("failed to get worker queue default options")
+	ErrInvalidWorkerQueueOption  = errors.New("worker queue option is invalid")
+	ErrCannotUpdateExistingQueue = errors.New("worker queue already exists")
+	ErrInvalidNodePool           = errors.New("node pool selection failed")
```

Comment:
```suggestion
	errWorkerQueueDefaultOptions = errors.New("failed to get worker queue default options")
	errInvalidWorkerQueueOption  = errors.New("worker queue option is invalid")
	errCannotUpdateExistingQueue = errors.New("worker queue already exists")
	errInvalidNodePool           = errors.New("node pool selection failed")
```

nitpick: ideally we would want to keep the variable local to the package unless specifically needed outside. Just to prevent them from being misused in the future

Reply:
These do not need to be exported as nothing outside uses them. Good catch.